docs/undergrad.yaml

```
@@ -434,7 +434,7 @@ Topology:
   Hilbert spaces:
     Hilbert projection theorem: 'exists_norm_eq_iInf_of_complete_convex'
     orthogonal projection onto closed vector subspaces: 'orthogonalProjection'
-    dual space: 'NormedSpace.instNormedSpaceDualToNormedFieldInstSeminormedAddCommGroupDual'
+    dual space: 'ContinuousLinearMap.toNormedSpace'
```

```suggestion
    dual space: 'NormedSpace.Dual'
```
seems more useful here

---

At first I was confused about whether we wanted the construction of the dual as type or a proof that the dual of a Hilbert space is a Hilbert space. 

But I guess the original was neither of these things?

---

I think the original thing probably meant the dual of a Hilbert space as a Hilbert space, so that you can state the Riesz representation theorem as an isometry of Hilbert spaces.

---

@PatrickMassot, do you prefer `ContinuousLinearMap.toNormedSpace` or `NormedSpace.Dual` here?

---

I think `NormedSpace.Dual` is better. Again the intended would be a combination of a Inner product space and a complete space instance, but we don't have a single declaration that would fit.